src/activity.ts

```diff
@@ -105,14 +105,15 @@ export const activity = async (
         (isExcluded(config.get(CONFIG_KEYS.Ignore.Workspaces)!, dataClass.workspaceFolder.uri.fsPath) ||
             isExcluded(config.get(CONFIG_KEYS.Ignore.Workspaces)!, dataClass.workspaceName));
 
-    const isNotInWorkspace = !isWorkspaceExcluded && (!dataClass.workspaceFolder || !dataClass.editor);
+    const isNotInWorkspace = (!isWorkspaceExcluded && (!dataClass.workspaceFolder));
+    const isNotInFile = (!isWorkspaceExcluded && !dataClass.editor);
```

Comment:
Added this if we do future things needing to tell if that it is not in a file / in a workspace independently